### ISSUE TITLE
Rewrite Readme

### ISSUE THREAD

**Issue by TheMysteriousVincent** [Saborknight/9liners-and-notepad#47]
I recently saw that our README.md file is deprecated. We should update it.
By that we also should restructure the readme.

**Comment by Saborknight** [comment]
Yeah, I was waiting until we have a release ready before doing that, no point in changing now if something changes in the future, requiring another Readme rewrite. Or that's what my thought on it was

**Comment by Saborknight** [comment]
Items which will need to be rewritten just before release of v1.0:

1. [readme.md](https://github.com/Saborknight/9liners-and-notepad/blob/999cd5348cdec3d481ee3c4ff4af234564024e51/readme.md)
1. Steam workshop description
1. [Author announcement](https://github.com/Saborknight/9liners-and-notepad/blob/999cd5348cdec3d481ee3c4ff4af234564024e51/addons/main/stringtable.xml#L14)

**Comment by Saborknight** [comment]
Since there's no tooltip to tell the user about this anymore, make sure to include a particular mention that left click enables controls, and right click closes the sheet entirely.
